seb_openedx/settings/test.py

```diff
@@ -18,6 +18,7 @@ class SettingsClass:
 SEB_COURSEWARE_MODULE = 'seb_openedx.tests.test_utils'
 SEB_COURSEWARE_INDEX_VIEW = 'seb_openedx.tests.test_utils'
 COURSE_KEY_PATTERN = r'(?P<course_key_string>[^/+]+(/|\+)[^/+]+(/|\+)[^/?]+)'
+COURSE_KEY_REGEX = r'(?P<course_key_string>[^/+]+(/|\+)[^/+]+(/|\+)[^/?]+)'
```

Comment:
`COURSE_KEY_PATTERN` is the same, why do we add this new one?

Reply:
Since I'm loading it from the settings it matches the name in edx-platform, but thanks to this comment I noticed that I copied the wrong definition.

Fixed now.